riscv/processor.cc

```diff
@@ -62,6 +59,10 @@ processor_t::processor_t(const isa_parser_t *isa, const char* varch,
   set_impl(IMPL_MMU_ASID, true);
   set_impl(IMPL_MMU_VMID, true);
 
+  xs_gatherer = new xs_gatherer_t(isa->get_max_xlen() == 32);
```

Comment:
Yes, this is a bug, and it should be fixed at `32c972`. In the following commit, I move all extensions related to the xs_gatherer, it can dynamically get the xlen information.